sensors/AS5048b.h

```diff
@@ -0,0 +1,105 @@
+/** @file AS5048b.h
+ *
+ * Copyright (c) 2021 IACE
+ */
+#ifndef AS5048B_H
+#define AS5048B_H
+
+#include <cstdint>
+#include "stm/hal.h"
+#include "stm/i2c.h"
+
+ /**
+  * Class describing a chain of AS5048B (i2c variant) Hall sensors
+  */
+class HallSensor {

```

Comment:
please call the class `AS5048` or `AS5048B` (what's the difference between them?)

i know the `AS5145` implements a `HallSensor`. we'll have to fix that.